paywall/src/paywall-builder/iframe.js

```diff
@@ -35,12 +35,18 @@ export function hide(iframe, document) {
   const width = '134px'
   const height = '160px'
   const collapsedMargin = '-104px'
+  let open = true
 
   // general settings
   document.body.style.overflow = ''
   iframe.style.backgroundColor = 'transparent'
   iframe.style.backgroundImage = 'none'
-  iframe.style['margin-right'] = collapsedMargin
+  iframe.style.marginRight = 0
+
+  setTimeout(() => {
+    open = false
+    iframe.style.marginRight = collapsedMargin
+  }, 2000)
```

Comment:
Quick nit: do you mind setting a constant for this? Maybe re-use SHOW_FLAG_FOR?

Reply:
sure